tests/framework/checks/expects.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package checks
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const (
+	workerLabel                                  = "node-role.kubernetes.io/worker"
+	descriptionTwoWorkerNodesWCPUManagerRequired = "at least two worker nodes with cpumanager are required for migration"
+)
+
+func ExpectAtLeastTwoWorkerNodesWithCPUManager(virtClient kubecli.KubevirtClient) {
+	ExpectAtLeastTwoNodes(GetWorkerNodesWithCPUManagerEnabled(virtClient), "")
+}
+
+// ExpectAtLeastTwoNodes uses gomega.Expect to verify that the given node list contains at least two elements. The
+// optionalDescription can be used to override the default description that notifies the reader of "not enough worker
+// nodes with cpumanager"
+func ExpectAtLeastTwoNodes(nodes []v1.Node, optionalDescription string) bool {
+	if optionalDescription == "" {
+		optionalDescription = descriptionTwoWorkerNodesWCPUManagerRequired
+	}
+	ginkgo.By("ensuring at least 2 worker nodes have cpumanager")
+	return gomega.Expect(len(nodes)).To(gomega.BeNumerically(">=", 2), optionalDescription)
+}
+
+func GetWorkerNodesWithCPUManagerEnabled(virtClient kubecli.KubevirtClient) []v1.Node {
+	ginkgo.By("getting the list of worker nodes that have cpumanager enabled")
+	nodeList, err := virtClient.CoreV1().Nodes().List(context.TODO(), v12.ListOptions{
+		LabelSelector: fmt.Sprintf("%s=,%s=%s", workerLabel, "cpumanager", "true"),
+	})
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	gomega.Expect(nodeList).ToNot(gomega.BeNil())
+	return nodeList.Items
+}
```

Comment:
I'm a little disturbed that we're following two different paths to achieve the same goal.
ref. https://github.com/kubevirt/kubevirt/blob/d7371feb56fcb138806b1b6e5b6ab154e900a2e0/tests/framework/checks/skips.go#L52-L75

But this is out of scope of this PR maybe!
I am thinking about a refactor about these

Reply:
Actually I don't really understand why at first we are asking whether the feature gate is there and then again ask for cpumanager being present, the former sounds superfluous to me. But yes, this is out of scope to me also.

Reply:
Well, actually it has sense to me! Even if the CPUManager FG is enabled, a node could not have the cpu manager running.

Reply:
Yes, that's why I think the FG check is superfluous, no?

Reply:
Yeah! My bad! Misread xD